src/common/TestDiscoveryVisitor.cs

```diff
@@ -1,26 +0,0 @@
-﻿using System.Collections.Generic;
```

Comment:
I had to remove this file because it started conflicting with https://github.com/xunit/xunit/blob/master/src/xunit.runner.utility/Visitors/TestDiscoveryVisitor.cs at the call sites.